packages/components/src/components/desktop/SpeakerStage.tsx

```diff
@@ -82,18 +83,31 @@ export const SpeakerStage = ({
         <div className={styles.stageCenter}>{mainView}</div>
         <div className={styles.sidebar}>
           {otherParticipants.map((participant) => {
-            return (
-              <ParticipantRenderer
-                key={participant.identity}
-                participant={participant}
-                width="100%"
-                aspectWidth={16}
-                aspectHeight={9}
-                showOverlay={showOverlay}
-                onMouseEnter={() => setShowOverlay(true)}
-                onMouseLeave={() => setShowOverlay(false)}
-              />
-            );
+            // @ts-ignore
```

Comment:
you should extract this into new Component because right now it's looks very messy